cpp_exercise/src/lib/ipcmsgqlib.cc

```diff
@@ -127,12 +125,11 @@ void MsgQueueReceiver::msgqTransfer(){
             _receive_size = mq_timedreceive(_msg_queue,(char*)buffer.data(),_msg_queue_msgsize,&_priority,&_ts);
```

Comment:
line 119 : using a sleep inside a loop is really expensive. Maybe you can try to avoid it.

Reply:
The reason I am using sleep to decrease the usage of CPU

Reply:
I don't understand why. The mq_timereceive will decrease the usage of the CPU while the process is waiting for a message and proceed if there is a message inside de queue.